backend/src/database/repositories/integrationRepository.ts

```diff
@@ -228,6 +228,24 @@ class IntegrationRepository {
     return this._populateRelations(record)
   }
 
+  static async findActiveIntegrationByPlatform(platform: PlatformType, tenantId: string) {
+    const options = await SequelizeRepository.getDefaultIRepositoryOptions()
+
+    const record = await options.database.integration.findOne({
+      where: {
+        platform,
+        tenantId,
+        deletedAt: null,
```

Comment:
AFAIK if we have `integration` sequelize model set to `paranoid` you don't have to specify `deletedAt: null` because it's automatic:
https://sequelize.org/docs/v6/core-concepts/paranoid/

Reply:
Cool, didn't know